petstore/src/main/scala/io/finch/petstore/model.scala

```diff
@@ -0,0 +1,77 @@
+package io.finch.petstore
+
+import argonaut.Argonaut._
+import argonaut._
+
+object model{
+
+  /*
+  Category case class for Pet object
+   */
+  case class Category(id: Long, name: String) //Not sure if this needs to be a class....
+
+  /*
+  Tag case class for Pet object
+   */
+//  case class Tag(id: Long, name: String)
+
+  /*
+  Status case class for Pet object
+   */
+//  case class Status(state: String)
+
+  /*
+    A pet object with the following fields:
+      id, category, name, photoUrls, tags, and status (available, pending, adopted)
+   */
+
+  case class Pet(
+      id: Long,
+      category: Category,
+      name: String,
+      photoUrls: Seq[String],
+      tags: Seq[String],
+      status: String
+      )
+
+  object Pet{
+    // Provides an implementation of the EncodeJson type class from Argonaut
+    implicit def profileEncoding: EncodeJson[Pet] = jencode2L(
```

Comment:
The number in the `jencode` methods should correspond to the number of fields in the case class (so in this case this should be `jencode6L`).